vpr/src/place/place.cpp

```diff
@@ -119,6 +119,16 @@ struct t_annealing_state {
     int move_lim;             // Current move limit
 };
 
+struct t_placer_timing_update_mode {
```

Comment:
You'll need a more detailed comment explaining each of these flags and how to set them.